### DISCUSSION TITLE
In-browser version?

### DISCUSSION THREAD

**Discussion by mercmobily** [eslint/eslint#14069] [General]
I remember checking, quite a while ago, whether eslint had a "browser" version. I would love to be able to validate code on my browser in an editor. Eventually, I would _love_ to be able to include eslint messages into CodeMirror 6 (which is being finalised).
CodeMirror 5 only has pre-packaged modules for jshint validation, which is not as comprehensive and configurable.

Basically, I would like to write a module for CodeMirror 6 so that I display eslint messages (warnings and errors).

Has client-side validation received some love? Is it a "thing" already?

**Comment by mdjermanovic** [comment]
Hi @mercmobily!

We don't have an official ESLint browser version, but [Linter](https://eslint.org/docs/developer-guide/nodejs-api#linter) class should probably work well in browsers for most use cases (we're using it for our [online demo](https://eslint.org/demo)). If something doesn't work, you can try with https://github.com/mysticatea/eslint4b

**Reply by mercmobily** [reply]
Thanks for the response! I am looking at the code, and it really looks like https://github.com/mysticatea/eslint4b is the way to go!
